### PR TITLE
Feature/use dp renderer

### PR DESCRIPTION
**Pull request by mwnicks**
### What

Set up to use new dp-renderer

### How to review

Ensure all tests still pass and the the service can be run.

### Who can review

Anyone but me

### CODE REVIEW

File: config/config_test.go [hunk 1]
```diff
@@ -25,7 +25,6 @@ func TestConfig(t *testing.T) {
 				So(cfg.HealthCheckInterval, ShouldEqual, 30*time.Second)
 				So(cfg.HealthCheckCriticalTimeout, ShouldEqual, 90*time.Second)
 				// TODO: Remove assertion once configuration variable has been removed from config.go
-				So(cfg.HelloWorldEmphasise, ShouldEqual, true)
```
Comment (rafahop) [inline config/config_test.go:28]: I think we should be adding assertions for the new config values here

Comment (mwnicks) [reply config/config_test.go:28]: Yes we should, good catch.

**Comment by rav-pradhan** [comment]
An adjacent point to this PR - it's worth double checking the secrets for the app to make sure it includes the `SITE_DOMAIN` value (set as the develop/prod site domains); we use this to determine whether or not a local version of the design system/Sixteens should be used (i.e., if the site domain is `localhost`, use local versions). If it's not there, you'll need to raise a PR to add that in too

**Review by rafahop — APPROVED** [review]
Looks good to me

**Pull request merged**